cmake/Toolchains/ios.cmake

```diff
@@ -12,12 +12,14 @@
 # PPSSPP platform flags
 set(MOBILE_DEVICE ON)
 set(USING_GLES2 ON)
-set(IPHONEOS_DEPLOYMENT_TARGET 6.0)
+set(IPHONEOS_DEPLOYMENT_TARGET 9.0)
```

Comment:
Is this the minimum supported version or the maximum?  Just asking to understand the impact.

-[Unknown]

Reply:
It's minimum supported version

Reply:
Min, plus I don't think any device runnion ios 6-8 could even run ppsspp.


![](https://dorianroy.com/blog/wp-content/uploads/2021/04/Screenshot-2021-04-24-at-18.53.59.png)